src/repror/cli/generate_html.py

```diff
@@ -94,9 +95,28 @@ def rerender_html(root_folder: Path, update_remote: bool = False):
 
     builds = get_rebuild_data()
 
+    # Statistics for graph
+    counts_per_platform = {}
+    # Get the last 10 days
+    start = datetime.now() - timedelta(days=9)
+    end_of_day = datetime(start.year, start.month, start.day, 23, 59, 59)
+    timestamps = [end_of_day + timedelta(days=i) for i in range(0, 10)]
     by_platform = defaultdict(list)
-
     for build in builds:
+
+        # Do it in the loop so that we do this only once per platform
+        # and we dont have to hardcode the platforms
+        if not counts_per_platform.get(build.platform_name):
```

Comment:
I think it's more pythonic to write this way:
`if build.platform_name not in counts_per_platform` as we can't have an empty value for key